lib/membrane/video_compositor/queue.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Membrane.VideoCompositor.Queue do
+  @moduledoc """
+  Define input pads and compositor core contracts, that each
+  implementation of queuing should meet.
```

Comment:
```suggestion
  Defines input pads and compositor core contracts, that each
  implementation of a queue should meet.
```
Alternatively, we can change `queueing` into `queueing strategies`